lib/order-sign.js

```diff
@@ -77,8 +76,15 @@ class SignHelper {
       }]
     }
 
-    const res = await this.client.api.transact(txdata, opts)
-    return this.fixTx(res)
+    let tx
```

Comment:
wouldn't it be more readable if we'd removed `else`? Anyway, it works as is so the final decision is up to you

```
if (this.client.ual) {
  const { transaction } = await this.client.ual.signTransaction(txdata, opts)
  return this.fixTx(transaction)
}

const res = await this.client.api.transact(txdata, opts)
return this.fixTx(res)
```

Reply:
yes, it will be removed once we remove the native scatter impl. completely